policyengine_us/parameters/gov/states/nj/tax/income/credits/property_tax/income_limit.yaml

```diff
@@ -0,0 +1,27 @@
+description: New Jersey limits the property tax credit to filers with federal adjusted gross income below this limit, based on filing status.  
+metadata:
+  period: year
+  unit: currency-USD  
+  label: New Jersey property tax credit income limit
+  reference:
+    - title: 2021 NJ-1040 Instructions - Property Tax Credit
+      href: https://www.nj.gov/treasury/taxation/pdf/other_forms/tgi-ee/2021/1040i.pdf#page=49
+    - title: 2022 New Jersey Revised Statutes Title 54A - New Jersey Gross Income Tax Act Section 54A:3A-20 - Credit instead of deduction
+      href: https://law.justia.com/codes/new-jersey/2022/title-54a/section-54a-3a-20/
```

Comment:
I dont see the 20,000 or 10,000 in the law, is there another law document that applies?